GitCommands/RevisionReader.cs

```diff
@@ -52,8 +52,7 @@ public sealed class RevisionReader : IDisposable
               /* Author email    */ "%aE%n" +
               /* Committer name  */ "%cN%n" +
               /* Committer email */ "%cE%n" +
-              /* Commit subject  */ "%s%n%n" +
-              /* Commit body     */ "%b" + EndOfBody;
+              /* Commit subject  */ "%B" + EndOfBody;
```

Comment:
Comment is now a little misleading. This is the body too, no?

Reply:
Will change to raw body

Reply:
Also updated a few other comments